docs/_articles/en/codebuilder/cb-uninstall.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Uninstall Code Builder
```

Comment:
@sbudhirajadoc - I suggest changing the title to "Remove Code Builder".  We talk about not having to install things with Code Builder.  I know we're installing a managed package but I think it might cause confusion